pkg/artifact/ingest_logs_test.go

```diff
@@ -231,3 +418,8 @@ func (w *testObjectWriter) Write(ctx context.Context, reader io.Reader, descript
 	w.gotArtifact = string(content)
 	return nil
 }
+
+// toPtr is a helper function to convert a type to a pointer of that same type.
+func toPtr[T any](i T) *T {
```

Comment:
nit: can you use our pointer package instead?

https://github.com/abcxyz/pkg/blob/main/pointer/pointer.go#L19

Reply:
done